src/main.cpp

```diff
@@ -1594,7 +1594,7 @@ bool GetTransaction(const uint256 &hash, CTransaction &txOut, uint256 &hashBlock
 }
 
 /** Return certificate in certOut, and if it was found inside a block, its hash is placed in hashBlock */
-bool GetCertificate(const uint256 &hash, CScCertificate &certOut, uint256 &hashBlock, bool fAllowSlow)
+bool GetCertificate(const uint256 &hash, CScCertificate &certOut, uint256 &hashBlock, bool fAllowSlow, int blockHeight)
```

Comment:
to keep the same logic used by GetTransaction, in case of no txindex, return the certificate only if it is not completely spent avoiding passing the blockHeight

Reply:
Done